README.md

```diff
@@ -22,6 +22,30 @@ For the latest version:
 
 - In Eclipse use the *Import Wizard* to import *Existing Projects into Workspace*
 
+
+
+Multi ScalaVersion support
+-------------------
+
+Since Scala IDE 4.0 multiple scala versions are supported in a single eclipse workspace. The Scala IDE uses by default the highest supported scala version, 2.11 since Scala IDE 4.0 and 2.12 since Scala IDE 4.6. The default can be overwritten at workspace level or at project level. The latter is stored in the file `.settings/org.scala-ide.sdt.core.prefs`. This file can be generated with sbteclipse. 
+
+**For sbteclipse 5.2.5 and up**
+
+Given
+  ```
+  EclipseKeys.defaultScalaInstallation := "xD.yD" // "2.12" is the default
```

Comment:
I don't understand what the `D` represents here

Reply:
D stands for "default". 'xD.yD' represents the scala installation eclipse is using as default.

Reply:
I updated the description. I hope it is more clear now.